DigitalOffice/Services/TimeManagementService/Models/CreateLeaveTimeRequest.cs

```diff
@@ -0,0 +1,14 @@
+﻿using LT.DigitalOffice.TimeManagementService.Enums;
+using System;
+
+namespace LT.DigitalOffice.TimeManagementService.Models
```

Comment:
I thought that we store requests in the RestRequest directory

Reply:
Now we do not have any agreements on models, except that they are in the models folder :(